nowplaying/options.py

```diff
@@ -77,6 +78,24 @@ def __init__(self):
             help="ticker XML output format",
             default="/var/www/localhost/htdocs/songticker/0.9.3/current.xml",
         )
+        self.__args.add_argument(
+            "--api-bind-address",
+            dest="apiBindAddress",
+            help="Bind address for the API server",
+            default="0.0.0.0",
+        )
+        self.__args.add_argument(
+            "--api-port",
+            dest="apiPort",
+            help="Bind port for the API server",
+            default=8080,
+        )
+        self.__args.add_argument(
+            "--api-auth-users",
+            dest="apiAuthUsers",
+            help="API Auth Users",
+            default={"rabe": "rabe"},
```

Comment:
hardcoding this isn't kewl
```suggestion
            default={"rabe": "rabe"},
```

Reply:
opened #290 so this doesn't block deploying this mvp